parlai/mturk/core/mturk_data_handler.py

```diff
@@ -405,17 +424,36 @@ def log_start_task(self, worker_id, assignment_id, conversation_id):
             conn.commit()
 
     def log_award_amount(self, worker_id, assignment_id, amount, reason):
-        """Update a pairing state to add a bonus to be paid, appends reason"""
+        """Update a pairing state to add a task bonus to be paid,
+        appends reason. To be used for automatic evaluation bonuses
+        """
         with self.table_access_condition:
             conn = self._get_connection()
             c = conn.cursor()
             reason = "${} for {}\n".format(amount, reason)
+            amount = int(amount * 100)
```

Comment:
maybe include an inline comment here explaining why you're scaling the amount 100x  😂 to a casual observer it looks magic

Reply:
Ah of course - don't want to mislead people into thinking we're just going to multiply bonuses by 100x 😂